var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -25,7 +25,7 @@ class Subversion(AutotoolsPackage):
     version('1.8.17', sha256='1b2cb9a0ca454035e55b114ee91c6433b9ede6c2893f2fb140939094d33919e4')
     version('1.8.13', sha256='17e8900a877ac9f0d5ef437c20df437fec4eb2c5cb9882609d2277e2312da52c')
 
-    variant('serf', default=True,  description='Serf HTTP client library')
+    variant('serf', default=False,  description='Serf HTTP client library')
```

Comment:
Curious.  What is the motivation for this change?  And does it apply in general?